flow/connectors/bigquery/qrep_avro_sync.go

```diff
@@ -309,17 +306,32 @@ func GetAvroType(bqField *bigquery.FieldSchema) (interface{}, error) {
 	case bigquery.FloatFieldType:
 		return considerRepeated("double", bqField.Repeated), nil
 	case bigquery.BooleanFieldType:
-		return "boolean", nil
+		return considerRepeated("boolean", bqField.Repeated), nil
 	case bigquery.TimestampFieldType:
-		return map[string]string{
+		timestampSchema := map[string]string{
```

Comment:
Should create types structs in qvalue for these avro types

Reply:
do you mean similar to how we have AvroSchemaArray? Got it